### ISSUE TITLE
Website to join or follow progress?

### ISSUE THREAD

**Issue by percramer** [SuddenDevelopment/WallPlottE#1]
Hi,

i was wondering if there is a website to join this project or to just follow the progress.

Regards,

Per

**Comment by SuddenDevelopment** [comment]
no. I'm not doing a website for this until it's far enough along for people to contribute to with some order. I plan on updating in the repo this weekend with updates. and some of the more detailed instructions in markdown files or wiki here in github.
